GOVERNANCE.md

```diff
@@ -0,0 +1,94 @@
+* [Great Contributors](#great-contributors)
+* [Collaborator nominations](#collaborator-maintainers-nominations)
+* [Lead Maintainers nominations](#lead-maintainers-nominations)
+* [Consensus seeking process](#consensus-seeking-process)
+
+<!-- /TOC -->
+
+## Lead Maintainers
+
+Neostandard Lead Maintainers are the organization owners.
+They are the only members of the `@neostandard/leads` team. The Lead
+Maintainers are the curators of the Neostandard project and their key responsibility
+is to issue releases of Neostandard and its dependencies.
+
+Lead maintainers are responsible for releases.
+
+## Collaborators
+
+Neostandard Collaborators maintain the projects of the Neostandard organization.
+
+Both Collaborators and non-Collaborators may propose changes to the source code
+of the projects of the organization. The mechanism to propose such a change is a
+GitHub pull request. Collaborators review and merge (_land_) pull requests
+following the [CONTRIBUTING](CONTRIBUTING.md) guidelines.
+
+### Collaborator activities
+
+* Helping users and novice contributors
+* Contributing code and documentation changes that improve the project
+* Reviewing and commenting on issues and pull requests
+* Merging pull requests
+
+The Lead Maintainers can remove inactive Collaborators or provide them with
+_Past Collaborators_ status. Past Collaborators may request that the Lead
+Maintainers restore them to active status.
+
+## Great Contributors
+
+Great contributors on a specific area in the Neostandard ecosystem will be invited
+to join this group by Lead Maintainers. This group has the same permissions of a
+contributor.
+
+## Collaborator nominations
+
+Individuals making significant and valuable contributions to the project may be
+a candidate to join the Neostandard organization.
+
+A Collaborator needs to open a private team discussion in [`neostandard/collaborators`](https://github.com/neostandard/collaborators) and list the
+candidates they want to sponsor with a link to the user's contributions. For
+example:
+
+* Activities in the Neostandard organization
+  `[USERNAME](https://github.com/search?q=author:USERNAME+org:neostandard)`
+
+Otherwise, a Contributor may self-apply if they believe they meet the above
+criteria by reaching out to a Lead Maintainer privately with the links to their
+valuable contributions. That Lead Maintainer will then open a private team discussion on
+GitHub regarding that and reply back when a decision has been made.
+
+The consensus to grant a new candidate Collaborator status is reached when:
+
+- at least one of the Lead Maintainers approve
+- at least two of the Team Members approve
+
+## Lead Maintainers nominations
+
+A Team Member may be promoted to a Lead Maintainers only through nomination by a
+Lead maintainer and with agreement from the rest of Lead Maintainers.
+
+## Consensus seeking process
+
+The Neostandard organization follows a Lazy [Consensus Seeking][] decision-making model,
```

Comment:
If we want to reference the OpenJS CPC version of this? https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-9-decision-making